tests/testthat/test-source.R

```diff
@@ -101,3 +101,7 @@ test_that("generate_include_paths handles paths with spaces", {
     expect_equal(generate_include_paths("cpp11"), "-I'/a path with spaces/cpp11'")
   }
 })
+
+test_that("", {
+
```

Comment:
Are we missing the tests for this?

Reply:
Also 🙏 in advance for writing a test for it!

Reply:
Wow I'm sooo good at writing tests :satisfied: